openedx/core/lib/xblock_builtin/xblock_discussion/static/js/URI_shim.js

```diff
@@ -1,8 +1,5 @@
 if (typeof URI === 'undefined') {
-    console.log('URI is not defined. Loading it now ...');
```

Comment:
@itsjeyd nit: actually it would be great to still have those log messages - it could help debugging when something goes wrong.